internal/testutil/evidencetest/evidence.go

```diff
@@ -0,0 +1,51 @@
+package evidencetest
+
+import (
+	"clouditor.io/clouditor/api/evidence"
+	"clouditor.io/clouditor/internal/testdata"
+	"clouditor.io/clouditor/internal/util"
+	"google.golang.org/protobuf/types/known/structpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+var (
+	MockListEvidenceRequest1 = &evidence.ListEvidencesRequest{
```

Comment:
I would prefer keeping these in the table tests directly, otherwise its very hard to see what kind of values the tests are using.

Reply:
The idea behind a separate file was the decoupling of test data. Might be the case that other tests need the data, this would lead to code duplication in my opinion.